tools/topology/sof/pipe-dai-playback.m4

```diff
@@ -20,3 +20,5 @@ W_PIPELINE(N_DAI_OUT, SCHEDULE_PERIOD, SCHEDULE_PRIORITY, SCHEDULE_FRAMES, SCHED
 
 P_GRAPH(DAI_NAME, PIPELINE_ID,
 	LIST(`		', `dapm(N_DAI_OUT, DAI_BUF)'))
+
+indir(`define', concat(`PIPELINE_PLAYBACK_SCHED_COMP_', PIPELINE_ID), N_DAI_IN)
```

Comment:
`N_DAI_OUT`

Reply:
@tlauda good catch. I've fixed it now. let me check if that fixes my problem

Reply:
@tlauda that fixed the problem. awesome. Thanks. I'm going to close this PR, clean up and re-submit.